perceval/backends/core/weblate.py

```diff
@@ -0,0 +1,374 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2020 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Valerio Cosentino <valcos@bitergia.com>
+#
+
+import logging
+
+import requests
+
+from grimoirelab_toolkit.datetime import str_to_datetime
+from grimoirelab_toolkit.uris import urijoin
+from ...utils import DEFAULT_DATETIME
+
+from ...backend import (Backend,
+                        BackendCommand,
+                        BackendCommandArgumentParser,
+                        DEFAULT_SEARCH_FIELD)
+from ...client import HttpClient, RateLimitHandler
+
+# Range before sleeping until rate limit reset
+MIN_RATE_LIMIT = 10
+# Default sleep time and retries to deal with connection/server problems
+DEFAULT_SLEEP_TIME = 1
+MAX_RETRIES = 5
+
+CATEGORY_CHANGES = "changes"
```

Comment:
Shouldn't it be better to call it `change`? That will mean we need to change the name of the variable also.

Reply:
The name of the category is `changes` why should we call it `change`?